web-marketplace/src/pages/ProfileEdit/ProfileEdit.Settings.tsx

```diff
@@ -38,10 +38,10 @@ export const ProfileEditSettings = () => {
   } = useLoginData();
 
   // Social providers
+  const socialProviders = useSocialProviders();
   const _socialProviders = socialProviders.map(p => ({
     name: p.name,
-    // TODO: we'll need to replace this with the email from the provider account #2211
-    email: privActiveAccount?.email,
+    email: privActiveAccount?.[`${p.id}_email`],
```

Comment:
Should we avoid using single-character variables even for temporary ones. Using `socialProvider` here might be more visible.